build.gradle

```diff
@@ -12,7 +12,7 @@ buildscript {
 	}
 
 	dependencies {
-		classpath 'com.android.tools.build:gradle:4.0.1'
+		classpath 'com.android.tools.build:gradle:4.0.2'
```

Comment:
I think there's some `4.2.x` already. Any reason not going there?

Reply:
Probably. I just followed what IntelliJ proposed me, but I'll have another look.

Reply:
The last version of IntelliJ IDEA does not yet support `4.2.x`. I would therefore prefer to stay on `4.1.x` for now.

Reply:
Fine with me, but you are on `4.0.x` at the moment.

Reply:
I force-pushed, so this is the outdated version. [Here](https://github.com/grote/Transportr/pull/755/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R15) is the new one.